### DISCUSSION TITLE
Php artisan tinker error with xdg

### DISCUSSION THREAD

**Discussion by Pseudoman21** [laravel/framework#33802] [General]
- Laravel Version: 7.24.0
- PHP Version: 7.4.8
- Database Driver & Version: 7.4.8

### Description:
php artisan tinker returns an error:

  Error

  Class 'XdgBaseDir\Xdg' not found

  at C:\xampppp\htdocs\tryLaravel\tryProject\vendor\psy\psysh\src\ConfigPaths.php:34
    30|      * @return string[]
    31|      */
    32|     public static function getConfigDirs()
    33|     {
  > 34|         $xdg = new Xdg();
    35|
    36|         return self::getDirNames($xdg->getConfigDirs());
    37|     }
    38|

  1   C:\xampppp\htdocs\tryLaravel\tryProject\vendor\psy\psysh\src\ConfigPaths.php:90
      Psy\ConfigPaths::getConfigDirs()

  2   C:\xampppp\htdocs\tryLaravel\tryProject\vendor\psy\psysh\src\Configuration.php:392
      Psy\ConfigPaths::getConfigFiles()

### Steps To Reproduce:
php artisan tinker

**Comment by GrahamCampbell** [comment]
Do you have more than one copy of tinker? ie one globally installed and one locally installed? Can you run `composer show` and `composer global show` and provide the output?
